constants/meetups.ts

```diff
@@ -1,4 +1,14 @@
 export const meetups = [
+  {
+    name: 'Leetcode Night: Code, Collaborate, Conquer!',
+    description: `Join us for Leetcode Night, where coders of all levels tackle Leetcode problems together. Whether you're prepping for interviews or just want to sharpen your skills, this is the perfect chance to collaborate, share strategies, and learn new techniques. Bring your laptop, team up, and let's solve some coding challenges! Any meetups will be posted on the Edmonton Unlimited meetup.com
+Ready to level up? See you at Leetcode Night!A combination of the YEGRB, Exchange.js, and Edmonton.py communities!`,
+    to: `https://www.meetup.com/edmontonunlimited/events/`,
```

Comment:
Also the organizer just messaged me and is hoping you could change the link. So sorry to make changes after you have already done the work. They would like the link to be changed to: https://discord.gg/hDGT9F9hbp